### PR TITLE
aura finance decoders

### PR DESCRIPTION
**Pull request by prettyirrelevant**
Related #7406 

A follow up PR will contain accountant and balances.

### CODE REVIEW

File: rotkehlchen/chain/evm/decoding/aura_finance/decoder.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.asset == self.evm_inquirer.native_token
+            ):
+                event.counterparty = CPT_AURA_FINANCE
+                event.event_subtype = HistoryEventSubType.FEE
+                # Calculate bridge fee, accounting for potential refunds
+                actual_bridge_fee = event.balance.amount if refund_event is None else event.balance.amount - refund_event.balance.amount  # noqa: E501
+                event.balance.amount = actual_bridge_fee
+                event.notes = f'Pay {actual_bridge_fee} {self.evm_inquirer.native_token.symbol} as bridge fee (to Ethereum)'  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/aura_finance/decoder.py:101]: why hard coded ethereum here?

File: rotkehlchen/tests/unit/decoders/test_aura_finance.py [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x6DF95c1F5AdD9cE98a013AD9809782d10272c6b8']])
+def test_aura_finance_deposit_arb(arbitrum_one_inquirer, arbitrum_one_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x9c3b996891eb0ffb5261486ff77c69c20a332499b9cf4196df7fab2c2123ea5f')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=arbitrum_one_inquirer, tx_hash=tx_hash)  # noqa: E501
+    user_address, timestamp, gas_str, deposit_amt, receive_amt, approve_amt = arbitrum_one_accounts[0], TimestampMS(1732504225000), '0.00000372964', '2.862546191448752712', '2.054508357973208982', '115792089237316195423570985008687907853269984665640564039448.345378216051229581'  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/decoders/test_aura_finance.py:22]: please don't use abbreviations like this. Write it here and elsewhere properly as "amount".

File: rotkehlchen/tests/unit/decoders/test_aura_finance.py [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+            balance=Balance(FVal(gas_str)),
+            location_label=user_address,
+            notes=f'Burn {gas_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/decoders/test_aura_finance.py:154]: Shouldn't it be a BRIDGE/FEE ?

Comment (prettyirrelevant) [reply rotkehlchen/tests/unit/decoders/test_aura_finance.py:154]: there's no BRIDGE/FEE. `BRIDGE` only exists as `HistoryEventSubType`

File: rotkehlchen/chain/evm/decoding/aura_finance/decoder.py (hunk 1 above)
Comment (prettyirrelevant) [reply rotkehlchen/chain/evm/decoding/aura_finance/decoder.py:101]: that's the only possible destination at the moment.

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**